setup.py

```diff
@@ -26,13 +26,17 @@
     include_package_data=True,
 
     classifiers=(
-        'Development Status :: 2 - Pre-Alpha',
+        'Development Status :: 5 - Production/Stable',
         'Environment :: Console',
+        'Framework :: Flask',
```

Comment:
Given that this is not a Flask plugin or Flask compatible package, but rather an application that _uses_ Flask, is this correct?